pkg/app/app.go

```diff
@@ -15,14 +18,21 @@ type AddCommandContext struct {
 	NoColor    bool
 	Dir        bool
 	Concurrent int
+	Filters    []string
 }
 
 //NewAddContextFromCli is a constructor which creates the context
 func NewAddContextFromCli(c *cli.Context) AddCommandContext {
+	filters := strings.Split(c.String("filter"), ",")
```

Comment:
So allowing multiple filters like this `--filter filterString1,filterString2`. Would it be stretch to support this as well `--filter filterString1 --filter filterString2`. I don't know `urfavcli` well enough to know if this a stretch.